src/main/java/teammates/common/util/Templates.java

```diff
@@ -367,6 +367,7 @@ public static String populateTemplate(String template, String... keyValuePairs)
             public static final String CONTRIB_TOOLTIPS_POINTS_RECEIVED =
                     "${Const.Tooltips.FEEDBACK_CONTRIBUTION_POINTS_RECEIVED}";
             public static final String CONTRIB_PARAM_STUDENT_NAME = "${Const.ParamsNames.STUDENT_NAME}";
+            public static final String EQUAL_SHARE_HELP = "${equalShareHelp}";
```

Comment:
Maybe add the `CONTRIB_` in front as well, since the concept of equal share only applies to contribution questions after all.